src/main/java/org/datadog/jmxfetch/reporter/StatsdReporter.java

```diff
@@ -52,6 +52,7 @@ private void init() {
         // When using UDS set the datagram size to 8k
```

Comment:
```suggestion
        // When using UDS set the datagram size to 8k and disable origin detection
```
We could update this comment since it does more than set the size to 8k now.